src/renderer/components/Header.vue

```diff
@@ -11,11 +18,20 @@
 import { Component, Vue, Prop } from 'vue-property-decorator'
 import { actions } from '../store/actions';
 
-@Component({ })
+@Component({
+  props: {
```

Comment:
I think, props property isn't necessary inside Component decorator in case of using @props.
see prop decorator implementation: https://github.com/kaorun343/vue-property-decorator/blob/master/src/vue-property-decorator.ts#L65